android/PhysicalWeb/app/src/main/res/values/strings.xml

```diff
@@ -75,6 +75,10 @@
     <string name="fatbeacon_notification_title">Physical Web is Sharing as a FatBeacon</string>
```

Comment:
Let's switch to lower case "sharing" in this title to be consistent with the wifi direct notification title.

(unrelated to this line): Further up in the file, the string fatbeacon_summary has "allows you to find beacons that contain**s**", should be "allows you to find beacons that contain"